two-regions-app-lb/README.md

```diff
@@ -0,0 +1,51 @@
+# Two Regions application behind LoadBalancer
+
+## Overview
+
+Using this template you can launch a simple application setup split over two regions and behind a load-balancer. You only need to enter your ssh key in the parameter
+section of the `masterstack.yaml` template.  
+The stack overview shows the FIP(s) as well as the security group assignment command example in the outputs section.
+
+![network topology](img/tworegionsapplbhorizonstackoutput.png)
+
+## Usage
+
+### Initial launch
+
+- Configure your ssh key in the parameter of the `masterstack.yaml` template.
+- Create a stack with this template: `openstack stack create -t masterstack.yaml <new stackName>`
+
```

Comment:
Here should be pointed out how the "openstack port set --security-group" command can be obtained and that (and why?) it must be executed (in the correct region!), otherwise that might be a hurdle for novice users.